aplib.net-demo/Assets/Scripts/MainMenu/CanvasManager.cs

```diff
@@ -40,13 +40,13 @@ public class CanvasManager : MonoBehaviour
     /// Name of the start screen.
     /// </summary>
     [SerializeField]
-    private string _sceneNameStartingMenu = "Settings"; // TODO:: Load main start screen
+    private const string _sceneNameStartingMenu = "Settings"; // TODO:: Load main start screen
 
     /// <summary>
     /// Name of the game scene
     /// </summary>
     [SerializeField]
-    private string _sceneNameGame = "InGameSettings"; // TODO:: Load main game screen
+    private const string _sceneNameGame = "WinCondition"; // TODO:: Load main game screen
```

Comment:
TODO's can be removed, I'd set the game scene default to 'GridSystem' now
```suggestion
    private readonly string _sceneNameGame = "WinCondition";
```

Reply:
Added readonly's and changed the scene strings to above.